02LessonTwo/homework/index.html

```diff
@@ -1,10 +1,45 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <title>Salvador Rodriguez Artwork</title>
   <link rel="stylesheet" href="css/style.css">
 </head>
+
 <body>
+  <!-- Defines page header -->
+  <div>
+    <img src="C:\Users\salva\OneDrive\Documents\GitHub\intro-workbook\02LessonTwo\homework\artwork1.jpg"/>
```

Comment:
You don't necessarily need to list the path from C: to 02LessonTwo; if it's in the same repository as homework, you can change this to "./artwork1.jpg"